spynnaker/pyNN/models/spike_source/spike_source_array_vertex.py

```diff
@@ -40,10 +40,7 @@ def _send_buffer_times(spike_times, time_step):
     if len(spike_times) and hasattr(spike_times[0], "__len__"):
         data = []
```

Comment:
Note this can be made an empty numpy array too if that helps

Reply:
NO because in this case data is a list of nparrays.

The issue was the mix between nparrays and empty lists confusing numpy

Reply:
Oh yes, sorry!